code/modules/projectiles/boxes_magazines/external_mag.dm

```diff
@@ -294,7 +294,7 @@
 		icon_state = "smg9mm-0"
 
 /obj/item/ammo_box/magazine/toy/smg/riot
```

Comment:
this no longer makse sense as these are now not riot ammo, secondly this type already exists as
/obj/item/ammo_box_magazine_toy/smg so your vending machine should just include that as a venderable item instead and remove this type path

https://github.com/tgstation/tgstation/blob/65bb0a3e32a5e8dfc22f7c9ea2df8312b4215455/code/modules/projectiles/boxes_magazines/external_mag.dm#L284

Reply:
in fact your vending machine doesn't appear to include these paths at all, so I'm not sure why you are doing this